analytics/agma/agma_module.go

```diff
@@ -261,6 +261,11 @@ func (l *AgmaLogger) LogVideoObject(event *analytics.VideoObject) {
 	l.bufferCh <- data
 }
 
+func (l *AgmaLogger) Shutdown() {
```

Comment:
Is it possible to add tests to cover these lines

Reply:
@AlexBVolcy updated the unit test for it